### DISCUSSION TITLE
How to copy files from the host to Kathará devices?

### DISCUSSION THREAD

**Discussion by P1c1s** [Kidel/Kathara#318] [Q&A] [answered]
I have a question about startup files. I'm trying to load some rules on iptables but doesn't work. I don't know if it's something that can be done. I wrote the following row in pc1.startup `iptables-restore < iptables.rules`. I have the iptables.rules file in the same directory of pc1.startup. Is there a way to check if the pc1.startup contain errors? Maybe log files? 
Thanks advanced.

**Comment by P1c1s** [comment]
I apologize, I did not notice that each terminal session initially has a log of the commands executed during configuration. In any case, my problem persists. I have this `iptables.rules: No such file or directory`. It seems to be a path problem.

**Comment by tcaiazzi** [comment, marked as answer]
Dear @P1c1s,  

To ensure the file is located within `pc1`, you need to create a directory named `pc1` in the lab. Once created, place the file inside that directory.  

During the device startup process, the file will automatically be copied into the `pc1` directory according to the specified path. For example, if you create the path `pc1/etc/test.txt`, the file `test.txt` will be copied into the `etc` directory within `pc1`.  

Please let me know if this helps.

Thanks!

**Reply by P1c1s** [reply]
Thanks for your quick reply. Now it works. Thank so much @tcaiazzi .
